### ISSUE TITLE
<@IF:[!$_GET['...'];!]> doesn't work on 1.3.6

### ISSUE THREAD

**Issue by sashabeep** [evolution-cms/evolution#231]
in 1.3.3 there was no problem
I am using this for simple banners displaying in lightbox modal window by link
```
<@IF:[!$_GET['path'];!]>
<iframe src="[!$_GET['path'];!]" width="[!$_GET['w']+10;!]" height="[!$_GET['h']+10;!]" frameborder="0"...ETC
<@ELSE>
	[!redir?id=`34`!] <---redirect to parent section
<@ENDIF>
```

Now despite the path to iframe and width/height parameters result page always redirecting me

**Comment by sashabeep** [comment]
[!$_GET['path'];!] on the page shows real GET value. Condition doesn't work

**Comment by yama** [comment]
Is this fixed?

**Comment by dmi3yy** [comment]
test this: <@IF:[!$_GET['path'];!]>[!$_GET['path'];!]<@ELSE>
get this: Parse error: syntax error, unexpected end of file in /manager/includes/document.parser.class.inc.php(1588) : eval()'d code on line 270

**Comment by sashabeep** [comment]
@dmi3yy try https://github.com/evolution-cms/evolution/issues/232 ? Maybe the same bug?

**Comment by yama** [comment]
```
<@if:[!$_GET['path'];!]>[!$_GET['path'];!]<@else>
```
- Write in capital letters
- It's syntax error. Close with <@ENDIF>
- ' ; ' Is unnecessary

```
<@IF:[!$_GET['path'];!]>[!$_GET['path'];!]<@ENDIF>
```
It is possible to improve it so as not to distinguish upper case letters and lower case letters. I'll consider.

**Comment by yama** [comment]
```
<@IF:[!$_GET['path'];!]>[!$_GET['path'];!]<@ENDIF>
```
Sorry, " ; " does no problem. Maybe other cause.

**Comment by yama** [comment]
> Write in capital letters
I write above. Maybe, Github change lowercase?

**Comment by yama** [comment]
```
<@IF:[!$_GET['path']!]>[!$_GET['path']!]<@ENDIF>
```
Is this already fixed?

**Comment by sashabeep** [comment]
Yep, seems like fixed 1.4 RC1
